test/integration/foreman_puppet/smartclass_parameter_js_test.rb

```diff
@@ -74,6 +74,8 @@ class SmartclassParameterJSTest < IntegrationTestWithJavascript
       within(:xpath, '//table') do
         click_link 'ssl'
       end
+      # see: https://projects.theforeman.org/issues/37251
+      page.driver.browser&.file_detector = nil
```

Comment:
Hmm, tests are failing now on:
```
NoMethodError: undefined method `file_detector=' for #<Selenium::WebDriver::Chrome::Driver:0x..fda95f2e33cc25c50 browser=:chrome_headless_shell>
    foreman_puppet/test/integration/foreman_puppet/smartclass_parameter_js_test.rb:78:in `block in <class:SmartclassParameterJSTest>'
rake aborted!
```

But why? There is the safe guard ...

Reply:
I don't know. I would propose to switch back to the version that worked.

Reply:
Yop, agree ... Sorry for the not working suggestion

Reply:
Nevermind. I thought it would work too. I updated the PR.